crt_portal/cts_forms/views.py

```diff
@@ -151,11 +154,23 @@ def get(self, request, id):
 
     def post(self, request, id):
         record = Report.objects.filter(id=id)
-        print(request.POST.get('next'))
+        report = record[0]
+
         updates = {}
+
```

Comment:
We might want to add a link to the docs here

```suggestion

        # see custom message options https://django-activity-stream.readthedocs.io/en/latest/data.html
```